chronosphere/tfschema/common.go

```diff
@@ -40,3 +40,25 @@ var MatcherListSchema = &schema.Schema{
 		},
 	},
 }
+
+// Used by both Monitor and SLO
+var MonitorSignalGrouping = &schema.Schema{
```

Comment:
Since we use this outside of monitors, let's make the name more general 
```suggestion
var SignalGrouping = &schema.Schema{
```